acpdr/swagger-specs/schema-registry.yaml

```diff
@@ -1386,20 +1516,20 @@ parameters:
     in: body
     schema:
       "$ref": '#/definitions/replaceClass'
-  mixinPayload:
+  fieldGroupPayload:
     name: payload
-    description: The JSON payload must include a 'meta:intendedToExtend' field listing the '$id' of the class(es) with which the mixin is compatible.
+    description: The JSON payload must include a 'meta:intendedToExtend' field listing the '$id' of the class(es) with which the field group is compatible.
     required: true
     in: body
     schema:
-      $ref: '#/definitions/createMixin'
-  replaceMixinPayload:
+      $ref: '#/definitions/createFieldGroup'
+  replaceFieldGroupPayload:
     name: payload
-    description: The JSON payload must include all of the fields that were originally required for creating the mixin, however this time with updated values. This includes an 'allOf' attribute referencing the $id of the class the schema will implement.
+    description: The JSON payload must include all of the fields that were originally required for creating the field group, however this time with updated values. This includes an 'allOf' attribute referencing the $id of the class the schema will implement.
```

Comment:
This sentence is a bit funky, with the whole "however this time with updated values", IMO.

I'd go with something simpler, like "... for creating the field group, but with updated values."

Additionally, I'd backtick `allof` and `$id`.